src/wp-includes/theme.php

```diff
@@ -188,39 +188,27 @@ function get_stylesheet() {
  *
  * @since 1.5.0
  * @since 6.4.0 Memoizes filter execution so that it only runs once for the current theme.
+ * @since 6.4.1 Memoization removed.
```

Comment:
```suggestion
 * @since 6.4.2 Memoization removed.
```

Should be next minor release?